python/lsst/dbb/buffmngrs/endpoint/finder/finder.py

```diff
@@ -273,6 +273,12 @@ def parse_rsync_logs(directory, blacklist=None,
     empty file ``<logfile>.<extension>`` which act as a sentinel preventing it
     from parsing the file again.
 
+    As the files from a single observation night can be placed in different
+    directories and these directories can be created in a time zone different
+    from the one the endpoint site operates in, be default the function
+    monitors also log files in directories corresponding to a day before and
```

Comment:
Why is it "a day before and a day after"?  Don't the past_days and future_days values determine how many days before and after?

Reply:
Yes, at the end of a day (no pun intended) the actual time span is determined by `past_days` and `future_days` settings, but I wrote
> **by default** the function [...]
I also see that I misspelled "by" (will fix), but I think you get my point.

Anyway, I'll add a sentence clarifying that this interval can be adjusted with `past_days` and `future_days`.